omod/src/main/java/org/openmrs/module/registrationapp/page/controller/FindPatientPageController.java

```diff
@@ -98,15 +100,16 @@ public void controller(UiSessionContext uiSessionContext,
         d.addColumn("encounterId", libraries.getDefinition(EncounterDataDefinition.class, "reporting.library.encounterDataDefinition.builtIn.encounterId"), "");
         d.addColumn("dateCreated", libraries.getDefinition(EncounterDataDefinition.class, "reporting.library.encounterDataDefinition.builtIn.dateCreated"), "");
         d.addColumn("encounterDatetime", libraries.getDefinition(EncounterDataDefinition.class, "reporting.library.encounterDataDefinition.builtIn.encounterDatetime"), "");
-
-        // add the paper record identifier, if the definition is available (provided by the paper record module)
-        PatientDataDefinition paperRecordIdentifierDefinition =  libraries.getDefinition(PatientDataDefinition.class, "paperrecord.patientDataDefinition.paperRecordIdentifier");
-        if (paperRecordIdentifierDefinition != null) {
-            model.addAttribute("paperRecordIdentifierDefinitionAvailable", true);
-            d.addColumn("paperRecordIdentifier", paperRecordIdentifierDefinition, "", new PropertyConverter(String.class, "identifier"));
-        }
-        else {
-            model.addAttribute("paperRecordIdentifierDefinitionAvailable", false);
+        
+        String gpPaperRecordIdDef = Context.getAdministrationService().getGlobalProperty(RegistrationAppConstants.GP_PAPER_RECORD_IDENTIFIER_DEFINITION);
+        model.addAttribute("paperRecordIdentifierDefinitionAvailable", false);
+        if (StringUtils.isNotBlank(gpPaperRecordIdDef)) {
+            // add the paper record identifier, if the definition is available (provided by the paper record module)
```

Comment:
let's change this to "if a definition is available (the default is provided by  the paper record module)